sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -3303,9 +3312,44 @@ SIREPO.app.directive('jobsList', function(requestSender, appState, $location, $s
             };
 
             appState.clearModels(appState.clone(SIREPO.appDefaultSimulationValues));
-            $scope.getJobs();
+            $scope.$on('$routeChangeSuccess', () => {
+                if ($location.path() == (SIREPO.APP_SCHEMA.route.admJobs)) {
```

Comment:
@moellep do you have a better way to do this than checking for routeChangeSuccess? The case this is handling is when we go to the adm-jobs page we want the jobs list to be refreshed on first load. W/o this feature the jobs list is empty and we have click the "refresh" button to load the list of jobs.